test/NLog.Extensions.Logging.Tests/CustomBeginScopeTest.cs

```diff
@@ -40,7 +40,7 @@ public void TestNonSerializableSayHi()
             var scopeString = runner.SayHi().Result;
             Assert.Single(target.Logs);
             Assert.Equal("Hi Earth. Welcome Earth People", target.Logs[0]);
-            Assert.Equal("Earth People", scopeString);
+            // Assert.Equal("Earth People", scopeString); <-- Bug https://github.com/aspnet/Logging/issues/893
```

Comment:
Dunno if this is solved yet, as the repos has been changed.

Reply:
Think it has been resolved in NetCore2.2, but that is not the long-term-support-version (NetCore 2.1 is)